utils/fmoconfig.py

```diff
@@ -187,7 +211,8 @@ def _get_vmdci(self, vmname: str) -> FMO_Schema:
 
 def get_fmoconfig_manager() -> FMOConfig_Manager:
     manager = FMOConfig_Manager("/etc/fmo-config.yaml",
-                                "/var/host/fmo-config.yaml")
+                                "/var/host/fmo-config.yaml",
+                                "/var/host/vmddp.conf")
```

Comment:
At least this needs to be read from `fmo-config.yaml` but let's make a ticket for that